src/api/ledgerManager.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+import {Ledger} from "../core/ledger/ledger";
+import type {LedgerLog} from "../core/ledger/ledger";
+import diffBy from "lodash.differenceby";
+
+/**
+ * Generic adaptor for persisting a JSON log to some storage backend
+ * (e.g. GitHub, local filesystem, a database, etc)
```

Comment:
docstring says it's a generic adapter for persisting a JSON log, but the interface actually only describes ledgers.